recipes/cutlass/build.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -ex
+
+# Acc. to https://cmake.org/cmake/help/v3.19/module/FindCUDAToolkit.html#search-behavior
+# CUDA toolkit is search relative to `nvcc` first before considering
+# "-DCUDAToolkit_ROOT=${CUDA_HOME}". We have multiple workarounds:
+#   - Add symlinks from ${CUDA_HOME} to ${BUILD_PREFIX}
+#   - Add ${CUDA_HOME}/bin to ${PATH}
+#   - Remove `nvcc` wrapper in ${BUILD_PREFIX} so that `nvcc` from ${CUDA_HOME} gets found.
+# TODO: Fix this in nvcc-feedstock or cmake-feedstock.
+# NOTE: It's okay for us to not use the wrapper since CMake adds -ccbin itself.
```

Comment:
I believe this was fixed upstream in cmake 3.20: https://gitlab.kitware.com/cmake/cmake/-/merge_requests/5777

Reply:
Thanks! Will try removing it. I have amassed a "toolkit" of workarounds if things don't work out of the box (mostly through the help of the ever-patient @jaimergp), and that one was one of the defaults - would be very happy to hear if it's obsolete now. :)

Reply:
Removed it, and it worked! :)